src/Components/ColorCircle.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+type PropType = {
+  style: any;
+  handleClick: () => void;
+  isActive: boolean;
+};
+
+const ColorCirle: React.FunctionComponent<PropType> = ({
+  style,
+  handleClick,
+  isActive,
+}) => {
+  if (isActive) {
+    return (
+      <button
+        className="color-circle"
+        style={style}
+        onClick={handleClick}
+      ></button>
+    );
+  }
+  return (
+    <button className="color-circle" style={style} onClick={() => {}}></button>
+  );
+};
```

Comment:
there is a prop in button for disabling it, we can use that here

Reply:
instead of conditional render, i added a disabled prop in the button